src/pudl/output/pudltabl.py

```diff
@@ -296,6 +302,11 @@ def _get_table_from_db(
                 "It is retained for backwards compatibility only."
             )
         table_name = self._agg_table_name(table_name)
+        logger.warning(
+            "PudlTabl is deprecated and will be removed from the pudl package"
+            "at the end of February 2024. To access the data returned by this method,"
+            f"use the {table_name} table in the pudl.sqlite database."
+        )
```

Comment:
True it will be noisy but I think it will make the migration easier for folks. The warning message spits out the table they should pull from instead of them having to look up the `PudlTabl` method in the spreadsheet. IDK, you're more familiar with how folks use `PudlTabl` so this warning might be super annoying.